lib/id/id-details.js

```diff
@@ -200,9 +200,8 @@ module.exports = function (options) {
 
 	const idfnIndexes = groundTruths.map((_, gtIndex) => binaryDistFn({gtIndex}, gtMatch[gtIndex]).fnIndexes);
 	const idfpIndexes = predictions.map((_, predIndex) => binaryDistFn(predMatch[predIndex], {predIndex}).fpIndexes);
-
-	const idtp1 = sum(groundTruths.map(p => p.filter(a => a).length)) - idfn;
-	const idtp2 = sum(predictions.map(g => g.filter(a => a).length)) - idfp;
+	const idtp1 = sum(groundTruths.map(p => p.filter(a => a!== null ).length)) - idfn;
```

Comment:
Xo rules space before `!==`

```suggestion
	const idtp1 = sum(groundTruths.map(p => p.filter(a => a !== null ).length)) - idfn;
```